charts/admission-controller/README.md

```diff
@@ -377,6 +377,18 @@ Either way, you should see some logs in Admission Controller tail
     {"level":"info","component":"scanning-evaluator","message":"denying container with name=nginx and image=nginx reason=\"Reject Always\""}
 
 
+## Running helm unit tests
```

Comment:
for the admission-controller the README is generated from the `README.tpl` with `make docs`, you need to add these changes to the template otherwise they would get overwritten in the future(the ci would probably fail too)

Reply:
thank you for the heads up